rllib/agents/trainer.py

```diff
@@ -735,48 +739,39 @@ def setup(self, config: PartialTrainerConfigDict):
         if self.config.get("log_level"):
             logging.getLogger("ray.rllib").setLevel(self.config["log_level"])
 
-        def get_scope():
-            if tf1 and not tf1.executing_eagerly():
```

Comment:
it seems the deterministic training test fails with tf1.enable_eager_execution() related error message.
we probably still need to call it here?

Reply:
Hmm, yeah, it's interesting. This is just a check here, but maybe it has an effect :/ I'll check.

Reply:
This should be fixed now. I had to move the global seeding after the enable_eager_execution call. Otherwise, tf assumes that it's not running in eager mode and complains when we try to enable it.

Reply:
@gjoliver ^

Reply:
Good thing it had nothing to do with the `get_scope`. The reason I got rid of it was that each policy (since 1.6) lives in its own Graph and has its own session. The Trainer itself should not carry any tf-graph anymore.

Reply:
sounds good, thanks for the info.